tcms/kiwi_auth/test_admin.py

```diff
@@ -109,3 +111,49 @@ def test_admin_can_add_new_users(self):
 
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertTrue(get_user_model().objects.filter(username='added-by-admin').exists())
+
+
+class TestGroupAdmin(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.tester.is_superuser = True
+        cls.tester.save()
+
+        cls.group = GroupFactory(name='NewGroupName')
+        cls.group.save()
+        cls.defaultGroups = Group.objects.filter(name__in=['Administrator', 'Tester'])
+
+    def test_should_not_be_allowed_to_change_groups_with_default_names(self):
+        for group in self.defaultGroups:
+            response = self.client.get(reverse('admin:auth_group_change', args=[group.id]))
+            self.assertNotContains(response,
+                                   '<input type="text" name="name" value="%s" class="vTextField"'
+                                   ' maxlength="150" required="" id="id_name">'
+                                   % group.name)
+            self.assertContains(response, '<div class="grp-readonly">%s</div>' % group.name)
+
+    def test_should_not_be_allowed_to_delete_default_groups(self):
+        for group in self.defaultGroups:
+            response = self.client.get(reverse('admin:auth_group_change', args=[group.id]))
+            self.assertNotContains(response, '<a href="%s" class="grp-button grp-delete-link">%s</a>'
+                                   % (reverse('admin:auth_group_delete', args=[self.group.id]), _('Delete')))
+
+    def test_should_be_allowed_to_create_new_group(self):
+        response = self.client.get(reverse('admin:auth_group_add'))
+        self.assertContains(response, '<h1>%s</h1>' % _('Add group'))
```

Comment:
The string marked for translation is incorrect, see `admin/change_form.html`, also possibly the grappelli conterpart of these templates.

Reply:
Still inforrect, see the string  `msgid "Add %(name)s"` in django/contrib/admin/locale/en/LC_MESSAGES/django.po